app/telegram/pylander.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+from dotenv import load_dotenv
+
+from app import config
+from .models import Bot
+
+load_dotenv()
+
+
+BOT_API = os.getenv("BOT_API", config.BOT_API)
```

Comment:
Maybe we should use FastAPI `Settings` object to do that?

Reply:
Yes, I'm going to create a Setting object.

Reply:
Done.